.cicd/test-package.run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+echo '+++ :minidisc: Installing EOSIO'
+
+if [[ $(apt-get --version 2>/dev/null) ]]; then # debian family
```

Comment:
Is this somehow better than simply performing `which apt-get`?  `which` returns success only if the target exists and is executable.

Reply:
It's functionally equivalent, this way may be slightly more verbose is the only thing.